service/controller/clusterapi/v31/resource/cleanupmachinedeployments/delete.go

```diff
@@ -21,7 +21,7 @@ func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
 
 	var machineDeployments []v1alpha1.MachineDeployment
 	{
-		r.logger.LogCtx(ctx, "level", "debug", "message", "finding MachineDeployments for tenant cluster")
+		r.logger.LogCtx(ctx, "level", "debug", "message", "finding machine deployments for tenant cluster")
```

Comment:
I don't really mind how to write it but I made it consistent within that resource now.